src/SFA.DAS.Forecasting.Application/AccountProjection/Services/AccountProjectionService.cs

```diff
@@ -32,5 +33,25 @@ public async Task<AccountProjectionExpiry> GetExpiringFunds(long expectedAccount
 
             return new AccountProjectionExpiry(expectedAccountId, projectionForDate.ProjectionCreationDate, expiryAmounts.OrderBy(c=>c.PayrollDate).ToList());
         }
+
+        public async Task<AccountProjectionSummary> GetProjectionSummary(long accountId, DateTime startDate, int numberOfMonths = 12)
+        {
+            var projections = await _repository.GetAccountProjectionByAccountId(accountId);
+
+            var projectionForDate = projections.FirstOrDefault();
+            if (projectionForDate == null)
+            {
+                return null;
+            }
+
+            IEnumerable<dynamic> modifiedProjections = projections.Select(x => new
```

Comment:
This can be var rather than IEnumerable<dynamic> the dynamic means we lose the strongly typed nature of C#. Changing this to var will return an IEnumerable<anonymous type> which will mean the code remains strongly typed

Reply:
Done